packages/launchpad/src/setup/ScaffoldedFiles.vue

```diff
@@ -79,7 +79,12 @@ const needsChanges = computed(() => props.gql.scaffoldedFiles?.some((f) => f.sta
 
 const mutation = useMutation(ScaffoldedFiles_CompleteSetupDocument)
 
-const completeSetup = () => {
-  mutation.executeMutation({})
+const completeSetup = async () => {
+  const { data } = await mutation.executeMutation({})
+
+  if (!data?.completeSetup?.currentProject?.isCTConfigured) {
+    // if config did not have time to load, we need to reload the page
+    window.location.reload()
```

Comment:
Is the project lifecycle manager not setup to notify us when the config as loaded? I wonder if there is a `pDefer` trick we could listen to in the `completeSetup` so as to get around this reload.

Reply:
It was not protecting as intended so I removed the culprit.